package.json

```
@@ -22,7 +22,7 @@
   "dependencies": {
     "@graphprotocol/graph-cli": "^0.81.0",
     "@graphprotocol/graph-ts": "^0.35.1",
-    "@requestnetwork/smart-contracts": "0.38.1-next.2071",
+    "@requestnetwork/smart-contracts": "^0.38.1-next.2082",
```

Please pin the version number :pray: 

```suggestion
    "@requestnetwork/smart-contracts": "0.38.1-next.2082",
```